source/server/admin/clusters_handler.cc

```diff
@@ -56,6 +80,13 @@ Http::Code ClustersHandler::handlerClusters(Http::ResponseHeaderMap& response_he
   return Http::Code::OK;
 }
 
+Admin::RequestPtr ClustersHandler::makeRequest(AdminStream& admin_stream) {
+  Buffer::OwnedImpl response;
+  ClustersParams params;
+  params.parse(admin_stream.getRequestHeaders().getPathValue(), response);
```

Comment:
Should we fail thereuest on invalid params? Or does ClusterParams maintain an invalid state (in which case add a comment here)

Reply:
Right now, ClusterParams always returns OK status because if the format isn't json, it will default to text format. I'm open to changing this behavior.